src/engine/Input/Keyboard.ts

```diff
@@ -6,6 +6,7 @@ import * as Events from '../Events';
  * Enum representing input key codes
  */
 export enum Keys {
```

Comment:
Since we are switching to `KeyboardEvent.code` we should switch this enum to be an enum of strings

```typescript
export enum Keys {
   Num1 = "Numpad1",
   A = "KeyA"
...
```

Reply:
you’re right. i’ll get this sorted out.